model/src/main/proto/spotlight.proto

```diff
@@ -0,0 +1,59 @@
+syntax = "proto3";
+
+package model;
```

Comment:
FYI that this proto will need to be added to the model module's BUILD.bazel file so that it can be included in Bazel builds of the app in later PRs.

Reply:
done